src/component/navigation/Navigation.jsx

```diff
@@ -0,0 +1,36 @@
+import {useContext} from "react";
+import PropTypes from "prop-types";
+
+import {NAVIGATION_ITEMS} from "../../tool/constant";
+import {NavigationContext} from "../../context/Navigation.context";
+
+import {ReactComponent as MenuIcon} from "../../icon/menu.svg";
+import "./navigation.scss";
+
+const Navigation = ({toggle, isToggled}) => {
+
+    const {nav} = useContext(NavigationContext);
```

Comment:
Rather define a new hook for this, so you can also check if the context is not defined and then throw an error. Just a good pattern follow for defensive coding using the context pattern